### PR TITLE
Adjust the header of the table listing the elections

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/nuancier/issues/79

### CODE REVIEW

File: nuancier/templates/elections_list.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <th>Year</th>
         <th>Submissions</th>
         <th>Votes open</th>
-        <th>Published</th>
+        <th>Votes closes / Results published</th>
```
Comment (puiterwijk) [inline nuancier/templates/elections_list.html]: I think you mean "Voting closes / Results published"?

**Comment by puiterwijk** [comment]
This is reasonable.
I would still prefer Voting opens/Voting closes though.

**Comment by ralphbean** [comment]
> I would still prefer Voting opens/Voting closes

Me too.

**Comment by puiterwijk** [comment]
:+1:

**Pull request merged**